getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,182 @@
+To continue, run:
+
+.. prompt:: bash
+
+   pipetask run -j 12 -b /repo/dc2 -d "skymap='DC2' AND tract=3829 AND patch=47" -i u/USERNAME/OUTPUT-COLLECTION-1 -o u/USERNAME/OUTPUT-COLLECTION-2 -p ApTemplate-DC2.yaml#makeTemplate --register-dataset-types
+
+This will also take some time.
+When it is complete, you should have good seeing coadds covering the entirety of patch 47 in tract 3829 for multiple bands and be ready to run the rest of the AP Pipeline (namely difference imaging and source association).
+
+Performing difference imaging to make an APDB
+---------------------------------------------
+
+This next step uses a second pipeline, which effectively includes :py:mod:`lsst.pipe.tasks.imageDifference.ImageDifferenceTask`, :py:mod:`lsst.ap.association.TransformDiaSourceCatalogTask`, and :py:mod:`lsst.ap.association.DiaPipelineTask`.
+
+.. prompt:: yaml
+
+   description: An AP pipeline for difference imaging with LsstCam-imSim
+
+   instrument: lsst.obs.lsst.LsstCamImSim
+   imports:
+   - location: ApTemplate-DC2.yaml
+     exclude:  # These tasks are not necessary, as we already have templates
+       - consolidateVisitSummary
+       - selectGoodSeeingVisits
+       - makeWarp
+       - assembleCoadd
+   - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+     exclude:  # These tasks come from the ApTemplate-DC2 pipeline instead
+       - isr
+       - characterizeImage
+       - calibrate
+
+This difference imaging pipeline uses the good seeing templates we built and treats all the DP0 defaults as input "science" images.
+
+**This is a two-step process.**
+First, create an empty sqlite APDB:
+
+.. prompt:: bash
+
+   make_apdb.py -c isolation_level=READ_UNCOMMITTED -c db_url="sqlite:////PATH-TO-DESIRED-APDB/ApPipeTest1.db"
+
+Second, run the pipeline:
+
+.. prompt:: bash
+
+   pipetask run -j 12 -b /repo/dc2 -d "skymap='DC2' AND tract=3829 AND patch=47" -i u/USERNAME/OUTPUT-COLLECTION-2,2.2i/defaults/DP0 -o u/USERNAME/OUTPUT-COLLECTION-3 -p ApPipe-DC2.yaml -c diaPipe:apdb.isolation_level=READ_UNCOMMITTED -c diaPipe:apdb.db_url="sqlite:////PATH-TO-DESIRED-APDB/ApPipeTest1.db" --register-dataset-types
+
+When this pipeline completes, you should have difference images and an APDB with populated tables (``DiaSource``, ``DiaObject``, etc.) for multiple bands in patch 47 of tract 3829 of this dataset.
+
+Processing Data with BPS
+========================
+
+The example data processing steps above assume a relatively small data volume (a single patch), so running from the command line and using an sqlite APDB is appropriate.
+However, if you want to process larger data volumes, you'll need to use the Batch Processing System (BPS, :py:mod:`lsst.ctrl.bps`) and a PostgreSQL APDB.
+
+Describing how to set up a PostgreSQL APDB is beyond the scope of this guide. One key difference between using an sqlite APDB versus a PostgreSQL APDB is that the former is a file on disk created from scratch when running ``make_apdb.py``.
```

Comment:
Could you provide a link to your PostgreSQL APDB notes here? Perhaps with a caveat that they are not meant to be a complete user guide.